scripts/release/index.js

```diff
@@ -38,6 +38,7 @@ cooker.cook('publish', [
       cook.pushTagToRemote,
       cook.createReleaseNotes(releaseNotesTemplate),
       cook.createGithubRelease,
+      cook.runNpmScript('deploy'),
```

Comment:
I think we need to install the devDependencies required for all this "shx", "run-p", "run-s", "gh-pages". Should be tried locally first with "npm run deploy" (eventually disconnecting network to test this)...

We could list the devDeps in "website/package.json" and this should update the main package.json file...